src/resources/js/tickets-attendees.js

```diff
@@ -58,6 +58,9 @@ var tribe_event_tickets_attendees = tribe_event_tickets_attendees || {};
 			}
 		});
 
+		$( '.trash' ).click( function ( e ) {
+			return confirm( Attendees.confirmation );
+		});
```

Comment:
Is the `.trash` element the link? If so, shouldn't it be `preventDefault()` first and check that the confirmation dialogue was confirmed before allowing the user to continue? What happens if you click on the Trash button but don't touch the confirmation dialogue for 10 seconds?

Reply:
.trash is the span element around the link: http://p.tri.be/Ax9C1C

We don't need `preventDefault()` here as we are using the native JS confirm that works same without using that. 

It won't continue if you don't click 'Ok', clicking cancel just cancels the action and the user is not redirected.

I have tried the 10-second delay and it won't do anything until an action button is pressed.

Reply:
So from the admin side, it may not be as important as the FE, but we still want to consider users that are not mouse users. So this would work if the user uses their mouse to click, but if the user were a keyboard user, this would not work.

Let's target the link instead to capture all the possible ways of interacting with the trash link.

Reply:
Thanks, @paulmskim. I have updated the target to link as you suggested. 

Before that, I tried to delete using the keyboard and it worked fine for me with Chrome using `esc` to cancel and `Enter` to confirm: http://p.tri.be/eeD8nM

I am not sure if you were referring to a similar use case like mine.